app/src/main/java/edu/uc/app/parkfinder/MainActivity.kt

```diff
@@ -11,8 +11,8 @@ class MainActivity : AppCompatActivity() {
         setContentView(R.layout.main_activity)
         if (savedInstanceState == null) {
             supportFragmentManager.beginTransaction()
-                    .replace(R.id.container, MainFragment.newInstance())
```

Comment:
Updated file styles according to: https://developer.android.com/kotlin/style-guide